### DISCUSSION TITLE
No owner or group shown in file manager properties

### DISCUSSION THREAD

**Discussion by womotu** [linuxdeepin/developer-center#3041] [Features Request & Ideas | 特性请求 & 头脑风暴]
## Describe the bug
Using Deepin file manager, properties of a file or folder doesn't include owner or group

## To Reproduce
1. In Deepin File Manager select a file or folder, right click and select Properties, then drop-down Permission Management.
2. Notice owner and group names are not provided here, or anywhere.

## Expected behavior
When viewing Permissions, it should include name of owner and name of group for the file or folder you're inspecting.

### Distro & Version
Linux Deepin 15.11

**Comment by meiyixiang** [comment]
Thank you for your feedback
